routes/reportRoutes.js

```diff
@@ -0,0 +1,7 @@
+const router = require('express').Router();
+const createReport = require('../controllers/createReport');
+
+//add report to the database
+router.post('/create', createReport);
```

Comment:
Would it be ok to change the endpoint to something like 'submit' instead of 'create'? because we will only send the report to the database if the user actually reviews and sends it right? (correct me if I'm missing something...)

Reply:
@MynahMarie, I think submit is more for a form in the front end, create means making a new collection in the database. This is for simple understanding for someone who might want to use the api even for a different project

Reply:
@hoslack ok then!

Reply:
so this is `/create` but what is sit creating so i would say add to the endpoint the thing you are creating cause it is not obvious